docs/reference/remap/expressions/assignment.cue

```diff
@@ -40,6 +40,21 @@ remap: expressions: assignment: {
 						.field = "value"
 						```
 						"""
+					"|=": """
+						Object merge assignment operator. Assigns the result of the right-hand side, merged with the left-hand side, to the left-hand side:
+
+						```vrl
+						.field |= {"foo": "bar"}
+						```
+
+						This is equivalent to using the `merge` function:
+
+						```vrl
+						.field = merge(.field, {"foo": "bar"})
+						```
```

Comment:
Just noting that it's also equivalent to using `.field = .field | {"foo": "bar"}`

Reply:
Aha, that is nifty, I didn't realize we supported that. I could see some collision if we add bitwise operators but I guess the operations are kinda similar.